modules/sdk-coin-avaxp/src/lib/utils.ts

```diff
@@ -274,18 +269,29 @@ export class Utils implements BaseUtils {
    * @param network required to stringify addresses
    * @return mapper function
    */
-  mapOutputToEntry(network: AvalancheNetwork): (TransferableOutput) => Entry {
-    return (output: TransferableOutput) => {
-      const amountOutput = output.getOutput() as any as AmountOutput;
-      const address = amountOutput
-        .getAddresses()
-        .map((a) => this.addressToString(network.hrp, network.alias, a))
-        .sort()
-        .join('~');
-      return {
-        value: amountOutput.getAmount().toString(),
-        address,
-      };
+  mapOutputToEntry(network: AvalancheNetwork): (Output) => Entry {
+    return (output: Output) => {
+      if ('getOutput' in output) {
+        // output is TransferableOutput
+        const amountOutput = output.getOutput() as AmountOutput;
+        const address = amountOutput
+          .getAddresses()
+          .map((a) => this.addressToString(network.hrp, network.alias, a))
+          .sort()
+          .join('~');
+        return {
+          value: amountOutput.getAmount().toString(),
+          address,
+        };
+      } else {
+        const evmOuput = output as EVMOutput;
+        return {
+          // it should be evmOuput.getAmount()
+          value: new BN((evmOuput as any).amount).toString(),
```

Comment:
not sure what this comment means. but why can't we use `evmOutput.getAmount()`?

Reply:
`evmOutput.getAmount()` return 0. 
I think that avalanchejs should fix it.

Reply:
https://github.com/ava-labs/avalanchejs/issues/652